symplyphysics/core/convert.py

```diff
@@ -1,11 +1,12 @@
 from sympy import Expr, sympify
 from sympy.physics.units import Quantity as SymQuantity
+from sympy.physics.units.quantities import PhysicalConstant
 
 from .dimensions import assert_equivalent_dimension
 from .symbols.quantities import Quantity
 
 
-def convert_to(value: Quantity, target_unit: SymQuantity) -> Expr:
+def convert_to(value: Quantity | PhysicalConstant, target_unit: SymQuantity) -> Expr:
```

Comment:
Using `SymQuantity` instead of `PhysicalConstant` looks more logical.

Reply:
I agree